tests/test_managers.py

```diff
@@ -53,6 +53,9 @@ def setUp(self):
         user = User.objects.create(username='foo')
         self.request = Request.objects.create(user=user, ip='1.2.3.4')
 
+    def tearDown(self):
+        Request.objects.all().delete()
```

Comment:
@felixxm: I don't think DB flush is needed, each test is inside a transaction. ([Ref](https://docs.djangoproject.com/en/1.9/topics/testing/overview/#writing-tests))